app/views/partials/_properties.html.erb

```diff
@@ -20,6 +20,11 @@
         question: "<%= t('aria.question') %>",
         section_heading: "<%= t('aria.section_heading') %>"
       },
+      errors: {
+        branches: {
+          unsupported_question: "<%= t('activemodel.errors.messages.unsupported') %>"
```

Comment:
if it's being grabbed from the locales can we therefore remove it from the data attribute on the div?

Reply:
Yes.
Sing it... dum, dum, dum... Another attribute bites the dust!

Reply:
PR should now have commit with removal of attr.